### ISSUE TITLE
No itemComponents in DestinyVendorResponse

### ISSUE THREAD

**Issue by bhollis** [Bungie-net/api#414]
`DestinyVendorResponse` (when called from `GetVendor`, at least), will not populate itemComponents even if you specify all the item component types in the request. I realize the API is still beta but I wanted to let you know. It'd be great to see item stats!

**Comment by vthornheart-bng** [comment]
Yes, for sure!  I actually just happened to fix this in our internal builds a few days ago, it should come out not during this week's release, but our next major release a couple weeks from now.

**Comment by bhollis** [comment]
❤️❤️❤️

**Comment by bhollis** [comment]
Nice, this is all working now!
